testrunner/app/controllers/testrunner.go

```diff
@@ -191,7 +192,25 @@ func describeSuite(testSuite interface{}) TestSuiteDesc {
 
 // errorSummary gets an error and returns its summary in human readable format.
 func errorSummary(err *revel.Error) (message string) {
-	message = fmt.Sprintf("%4sStatus: %s\n%4sIn %s", "", err.Description, "", err.Path)
+	EXPECTED_PREFIX := "(expected)"
```

Comment:
Code of `testrunner` has already been `lint`ed. Could you, please, follow Golang's naming conventions. Read more about them here: https://github.com/golang/go/wiki/CodeReviewComments